script/hassfest/icons.py

```diff
@@ -76,7 +76,7 @@ def icon_schema_slug(marker: type[vol.Marker]) -> dict[vol.Marker, Any]:
         }
     )
 
-    if integration_type == "entity":
+    if integration_type in ("entity", "helper", "system"):
```

Comment:
Helpers can provide both entities on their own domains or provide normal entities like any other integration does (e.g., groups, mold, trend).

For helpers we need to extend to support both schemes